packages/-ember-caluma/app/templates/docs/index.md

```diff
@@ -21,39 +21,23 @@ order to support [fragments on unions and interfaces](https://www.apollographql.
 Create a new service `app/services/apollo.js` and extend the apollo service with
 the provided mixin:
 
-{{docs-snippet name='usage/apollo.js'}}
```

Comment:
How come the path change from `snippets/usage` to `snippets`?

Reply:
We had a folder `snippets/config` and `snippets/usage`. Since the config snippet was deleted, I moved all the usage snippets to the top level snippet folder.

Reply:
Ahh, ok :+1: